hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/callback/TestChangeConfigInitCallback.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.callback;
+
+import org.apache.hudi.client.BaseHoodieClient;
+import org.apache.hudi.config.HoodieWriteConfig;
+
+import org.apache.avro.Schema;
+
+import static org.apache.hudi.config.HoodieWriteConfig.WRITE_SCHEMA_OVERRIDE;
+
+/**
+ * A test {@link HoodieClientInitCallback} implementation to add the property
+ * `user.defined.key2=value2` to the write schema.
+ */
+public class TestChangeConfigInitCallback implements HoodieClientInitCallback {
```

Comment:
naming is bit confusing w/ other test classes in general. may be we did not come up w/ a nomenclaure before. but can we start following some from now on. 
may be ChangeConfigInitCallbackTest to differentiate regular test classes from 
test implementations used for testing purpose

Reply:
I renamed the classes with `TestClass` suffix to differentiated them from other classes containing the actual tests.